### PR TITLE
NOISSUE: update VCallRequest in CTMethod tests

### PR DESCRIPTION
**Pull request by ElSacramento**
**- What I did**

**- How I did it**

**- How to verify it**

**- Description for the changelog**

### CODE REVIEW

File: ledger-core/v2/virtual/integration/method_test.go [hunk 1]
```diff
@@ -323,11 +323,11 @@ func TestVirtual_Method_WithExecutor(t *testing.T) {
 			callResultPl := metaPayload.(*payload.Meta).Payload
 			callResultPlType, err := payload.UnmarshalType(callResultPl)
 			assert.NoError(t, err)
-			assert.Equal(t, payload.TypeVCallResult, callResultPlType)
+			assert.Equal(t, payload.TypeVStateRequest, callResultPlType)
```
Comment (bigbes) [inline ledger-core/v2/virtual/integration/method_test.go:326]: I've wrote a bad test, we need to prepare an object, before (so no VStateRequest is needed)
Later we should obtain VCallResult

Comment (ElSacramento) [reply ledger-core/v2/virtual/integration/method_test.go:326]: `Method_PrepareObject` doesn't prepare an object?

Comment (ElSacramento) [reply ledger-core/v2/virtual/integration/method_test.go:326]: Or I can use it?

Comment (ElSacramento) [reply ledger-core/v2/virtual/integration/method_test.go]: done

**Pull request merged**